web/src/features/charts/FuturePrice.tsx

```diff
@@ -176,9 +179,17 @@ export function PriceBar({
 }
 
 function PriceDisplay({ price, currency }: { price: number; currency: string }) {
+  const symbol = getSymbolFromCurrency(currency);
+  if (symbol == '€') {
+    return (
+      <p className="min-w-[66px] text-nowrap text-end text-sm font-semibold tabular-nums">
+        {`${price} ${symbol}`}
+      </p>
+    );
+  }
   return (
     <p className="min-w-[66px] text-nowrap text-sm font-semibold tabular-nums">
-      {`${price} ${getSymbolFromCurrency(currency)}`}
+      {`${symbol}${price}`}
     </p>
```

Comment:
@Alportan do you think we can put the currency symbol on the right side for all currencies? I think most would would have it on the right, especially if we start using more local currencies in the future?

We might even want to consider switching to named currencies like `USD`, `EUR`, `SEK`, `DKK` etc which should all be placed on the right.

Reply:
> @Alportan do you think we can put the currency symbol on the right side for all currencies? I think most would would have it on the right, especially if we start using more local currencies in the future?

Tony commented on the fact that it was on the right side, and I think many english speakers would notice it. So I don't think that we should put it on the right side for all currencies.

> We might even want to consider switching to named currencies like `USD`, `EUR`, `SEK`, `DKK` etc which should all be placed on the right.

I think that depends on the language, in english you put it in front, that is what [this](https://www.scribens.com/grammar-rules/typography/currency-format.html) article says at least

Reply:
I think you're actually supposed to write the currency code on the left always and where you put the symbol depends on the language: [currency symbols and codes](https://proofed.com/knowledge-hub/currency-symbols-and-codes/)

Reply:
Gotta love localisation 🙃

Reply:
Here are the more in depth documentation around the various options: https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Intl/NumberFormat/NumberFormat#currency_2